edx-platform/pearson-bsg-theme/lms/templates/certificates/_accomplishment-header.html

```diff
@@ -0,0 +1,6 @@
+<!-- Style attached in this file because it is outside lms/static/sass/ and it in not being applied when overridden -->
```

Comment:
```suggestion
<!-- Style attached in this file because it is outside lms/static/sass/ and it is not being applied when overridden -->
```
Minor grammar. @ivanvgh Please apply it to the other files.

Reply:
Done.